### PR TITLE
Copy weight files into node-template

### PR DESCRIPTION
**Pull request by tifecool**
PR attempts to solve issue #11090 

Copied weight files and TOML from support.

polkadot address: 1s51dYXziQwTnSnModErKfra3TeEsVipqzY846CLCTEXtHJ

### CODE REVIEW

**Comment by ggwpez** [comment]
Thanks for the help!  
You still need to use the weights, as currently the node is still using the weights from `frame_support`.  
I did not mark this issue as "mentor" since there is more involved to what I read and it was more of a reminder for me.  

Could you please rename the package to `node-template-runtime-constants`? As more constants will be added there analogous to Polkadot.

**Comment by tifecool** [comment]
I'm on it. Sorry about the late response.

**Comment by ggwpez** [comment]
That one point is still outstanding:  
> You still need to use the weights, as currently the node is still using the weights from frame_support.

I think we can also merge as-is and then I do that together with an update of the weights.

**Comment by tifecool** [comment]
> I think we can also merge as-is and then I do that together with an update of the weights.
I'd like this. Not sure I fully understand the code so well and would love to be involved 🙏

**Comment by shawntabrizi** [comment]
@tifecool if you would like a small tip in KSM or DOT, please include your address in the PR description

**Comment by tifecool** [comment]
> @tifecool if you would like a small tip in KSM or DOT, please include your address in the PR description

done ❤️

**Comment by ggwpez** [comment]
I thought about this again and think that it is better to not create a new crate, since we somehow need to keep the balance between completeness and understandability.  
On the one hand it should be a complete example but on the other hand a new user should have a chance at understanding it.  
So in the current state the weights are in their own folder besides the runtime `lib.rs`.

File: bin/node-template/runtime/src/lib.rs [hunk 1]
```diff
@@ -43,6 +40,13 @@ pub use sp_runtime::{Perbill, Permill};
 /// Import the template pallet.
 pub use pallet_template;
 
+// Use own values for `BlockExecutionWeight`, `ExtrinsicBaseWeight`, `RocksDbWeight`
+// and `ParityDbWeight` instead the defaults from [`frame_support::weights`].
+pub mod weights;
+
+// Re-export the non-default weights.
+pub use weights::*;
```
Comment (shawntabrizi) [inline bin/node-template/runtime/src/lib.rs:48]: Id rather we dont have this re-export for clarity where things are coming from

Comment (tifecool) [reply bin/node-template/runtime/src/lib.rs:48]: Maybe we could rename the weights.rs to template_weights.rs instead?

**Comment by shawntabrizi** [comment]
In general I don't think the node-template needs custom weight stuff. It adds quite a bit of complexity, and doesn't really make sense since users are likely to change the node template right away, and thus invalidate the weights.

Probably best to just get the default weights from the main substrate node at this time.

**Pull request closed without merging**